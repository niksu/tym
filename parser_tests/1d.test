hellom1123_AS12_1(a, B, a123) :- a(), b().
c() :- a(). a(b).
