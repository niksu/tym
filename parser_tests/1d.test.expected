input contents |hellom1123_AS12_1(a, B, a123) :- a(), b().
c() :- a(). a(b).
|
(no query given)
stringed file contents (size=148, remaining=2852)
|predicate_hellom1123_AS12_1(constant_a, B, constant_a123) :- predicate_a(), predicate_b().
predicate_c() :- predicate_a().
predicate_a(constant_b).|
stringed file contents strlen=147
